sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -66,5 +67,18 @@ export function useFormatters() {
     return 0;
   };
 
-  return { dateString, dateOnlyLongString, getDatesDiff };
+  const getFormattedAddress = (address: Address): string => {
+    const formattedAddress = [];

```

Comment:
_formattedAddress_ is declared as an Array of any. Please change it to string.
![image](https://user-images.githubusercontent.com/61259237/142302396-16d1bd7a-84eb-4318-9c5e-6097bdd20170.png)